Client/Connection.ts

```diff
@@ -19,11 +19,15 @@ export class Connection {
 		path: string,
 		method: string,
 		request?: any,
-		parameters?: Record<string, any>
+		parameters?: Record<string, any>,
+		multipart?: boolean,
+		formData?: FormData
```

Comment:
the boolean isnt needed if you have the formdata already. but you don't need it, because request? is already any, you could check its type instead of having a separate argument